src/bots/monitor/index.js

```diff
@@ -0,0 +1,85 @@
+import client from '../../client';
+import { CHANNEL_NAMES, defaultStrings } from './constants';
+import {
+  attachmentIsAllowed,
+  deleteAndReturnMessage,
+  getChannelIdForName,
+  getChannelNameForId,
+  getChannelPostTitleForName,
+  isMessageReply
+} from './utils';
+
+client.on('messageCreate', async (message) => {
+  const { author, channelId } = message;
+
+  // Run routine based on id
+  switch (channelId) {
+    case getChannelIdForName(CHANNEL_NAMES.CAREER_QUESTIONS):
+    case getChannelIdForName(CHANNEL_NAMES.HIRING):
+    case getChannelIdForName(CHANNEL_NAMES.SHARE_YOUR_CONTENT):
+      {
+        // <=== VALIDATION ===>
+        // Get the channel's name
+        const channelName = getChannelNameForId(channelId);
+        if (channelName === null) return;
+
+        // Do not allow top-level messages as replies
+        if (isMessageReply(message)) {
+          author.send(defaultStrings.invalidReply(channelName));
+          deleteAndReturnMessage(message);
+          return;
+        }
+
+        // <=== CREATE NEW THREAD ===>
+        // Create a new thread under each message sent
+        const postTitle = getChannelPostTitleForName(channelName);
+        message.startThread({
+          name: defaultStrings.threadTitle(author.username, postTitle)
+        });
+      }
+      break;
+
+    // Resume review channel
+    case getChannelIdForName(CHANNEL_NAMES.RESUME_REVIEW):
```

Comment:
I get the reasoning behind all this, but referring to my more dynamic suggestion I think this is a single breakaway case that can be dealt with like this,